docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -2,488 +2,716 @@
-  [19:31:06 INFO    handler] processing: Verb=set Name=BKaiql Value=94175 address=1cf12669cbc17d076a1accb4b0bb61f40ed4f999173b90e3ca2591875a55fee2947661e60fa1c57b41ef0f2660176b945a01c85ff645543297068a3fb1306324a19612
-  [19:31:06 INFO    handler] processing: Verb=set Name=wpMQmE Value=47316 address=1cf1260f6bdf66b65ff7c00ec58c4deccffd167bfee7a85698880dfa485df3de1ec18a5b2d1dc12849743d1c74320108360a2d40d223b35fbc1c4ea03bbd8306480c62
-  [19:31:06 INFO    handler] processing: Verb=set Name=GTgrvP Value=31921 address=1cf12606ac7db03c756133c07d7d02b59f3ef9eae6774fe59c75c88ab66a9fabbbaef9975dbf9aa197d1090ed126d7b18e2
+#. Run the following command:
 
+   .. code-block:: console
 
-Viewing Blocks and State
-========================
+      user@client$ ps aux | grep [s]awtooth-rest-api
+      sawtooth  2829  0.0  0.3  55756  3980 pts/0    S+   19:36   0:00 sudo -u sawtooth sawtooth-rest-api -v
+      sawtooth  2830  0.0  3.6 221164 37520 pts/0    Sl+  19:36   0:00 /usr/bin/python3 /usr/bin/sawtooth-rest-api -v
+
+#. If necessary, run the following command to restart the REST API.
+
+   .. code-block:: console
+
+      user@client$ sudo -u sawtooth sawtooth-rest-api -v
+
+
+Step 7: Configure the Transaction Family Settings
+=================================================
+
+By default, Sawtooth allows transactions from any transaction family. In this
+step, you will change the configuration settings to allow only IntegerKey and
+Settings transactions.
+
+This configuration change is itself a transaction that is submitted in a batch.
+Sawtooth configuration settings are stored on the blockchain so that all
+validator nodes in a Sawtooth network know the configuration. The setting for
+the list of supported transaction families is defined by the
+:doc:`Settings transaction family
+<../transaction_family_specifications/settings_transaction_family>`.
+
+#. Use the following ``sawset proposal create`` command to create and submit the
+   new settings to the validator (as a batch of transaction).
+
+   .. code-block:: console
+
+      user@client$ sawset proposal create sawtooth.validator.transaction_families='[{"family": "intkey", "version": "1.0"}, {"family":"sawtooth_settings", "version":"1.0"}]'
+
+   This command specifies a JSON array that tells the validator to accept only
+   transactions from version 1.0 of the IntegerKey and Settings transaction
+   processors, using the family names ``intkey`` and ``sawtooth_settings``,
+   respectively.  For more information on transaction family names, see
+   :doc:`../transaction_family_specifications`).
+
+#. Examine the output in the validator terminal window. The logging output
+   includes a ``TP_PROCESS_REQUST`` message, as in this example:
+
+      .. code-block:: console
+
+         [2018-03-14 16:24:49.587 DEBUG     received message of type: TP_PROCESS_REQUEST
```

Comment:
This seems like an odd message to use as an indication that the batch was received. It might be better to direct the reader to the settings-tp terminal window instead, where they would see the transactions run, or to show the rest api submitting the batch to the validator with `[2018-03-15 14:21:38.058 DEBUG    route_handlers] Sending CLIENT_BATCH_SUBMIT_REQUEST request to validator`